### ISSUE TITLE
Additions to Particle Attractor

### ISSUE THREAD

**Issue by iMobCoding** [mob-sakai/ParticleEffectForUGUI#321]
It would be nice if PA can have some randomness for Delay Rate like normal particles do: constant, between 2 constants, curve & between 2 curves. Justo to break its uniformity when attracting particles...

Thanks

**Comment by mob-sakai** [comment]
Thank you for your reporting!

**Comment by mob-sakai** [comment]
For example, in the demo, I set `StartLifetime=[5-8]` to add randomness.

![UIParticle_Demo_-_ParticleEffectForUGUI_v4_-_Windows__Mac__Linux_-_Unity_2021_3_30f1__Metal__compressed](https://github.com/user-attachments/assets/9d373f42-798a-4879-9bd7-2a323110f24b)

![Aug-08-2024 02-06-34_compressed](https://github.com/user-attachments/assets/170d1782-b431-4204-b18b-14fa4a79a49d)

**Comment by iMobCoding** [comment]
Yes I know, it's one way to add randomness. But if we, for example, add the curve in the attractor, we can then join Delay, Max Speed and Movement into the one property. In one curve you set when particle starts going to the attractor, its moving speed and Ease. And adding 2 curves there would achieve randomness too. That way we can avoid my first suggestion (constant and 2 constants) as all of that could be achieved with curves

**Comment by mob-sakai** [comment]
Maybe using `Particle.randomSeed` will allow you to fix the curve for the particles.
